classes/Repository/ModuleRepository.php

```diff
@@ -63,6 +64,19 @@ public function getUpgradeLink()
         ]);
     }
 
+    /**
+     * @return string
+     */
+    public function getActiveLink()
```

Comment:
```suggestion
    public function getEnableLink()
```

So you match the action name